src/main/resources/assets/computercraft/lua/rom/apis/paintutils.lua

```diff
@@ -62,7 +62,7 @@ function drawLine( startX, startY, endX, endY, nColour )
         return
     end
     
-    local minX = math.min( startX, endX )
+    local minX, maxX, minY, maxY = math.min( startX, endX )
```

Comment:
this is not a nice line of code! If the intent is to assign one variable and declare 3 others, put them on new lines!